modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -62,13 +65,33 @@ The products you need for this procedure are:
 .ES Products - WebUI
 |===
 | OS Version | Product Name
-| {es} 6     | SUSE Linux Enterprise Server with Expanded Support 6 x86_64
-| {es} 7     | SUSE Linux Enterprise Server with Expanded Support 7 x86_64
-| {es} 8     | RHEL or SLES ES or CentOS 8 Base
+
+| {es} 6
+| SUSE Linux Enterprise Server with Expanded Support 6 x86_64
+
+| {es} 7
+| SUSE Linux Enterprise Server with Expanded Support 7 x86_64
+
+| {es} 8
+| RHEL or SLES ES or CentOS 8 Base and
+  SUSE Linux Enterprise Server with Expanded Support 8 x86_64
 |===
 
 include::snippets/add_channels_wizard.adoc[]
 
+This procedure will also create the Tools Channels containing the software required to work with {productname}.
+
+[[es-channels-tools]]
+[cols="1,1,1", options="header"]
+.ES Tools Channels
```

Comment:
@Loquacity please check! I changed the names and anchors. Not sure if they are used somewhere else. But the old names were just wrong

Reply:
fixed the table syntax and a couple of minor line edits.